### PR TITLE
[PhpUnitBridge] run PHPUnit in the same process

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #21083
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by xabbuh** [comment]
@julienfalque Can you check if this change solves the issue you described in https://github.com/symfony/symfony/pull/21083#issuecomment-269696733 too?

**Comment by nicolas-grekas** [comment]
This currently breaks the handling of the exit code (see var $exit). Otherwise I think that's a good way to go. Maybe see if we can do something with OK/KO in a shutdown function ? Or deal with it in .travis.yml file as it was done before.

**Comment by xabbuh** [comment]
Unfortunately, we cannot detect the exit code inside a shutdown function. So we will need to update our Travis CI config after this has been merged.

File: src/Symfony/Bridge/PhpUnit/bin/simple-phpunit [hunk 1]
```diff
@@ -180,27 +180,11 @@ if ($components) {
...
-        stream_copy_to_stream($pipes[1], STDOUT);
-        fclose($pipes[1]);
-        $exit = proc_close($proc);
-
-        readfile($errFile);
-        unlink($errFile);
-    }
+    include "$PHPUNIT_DIR/phpunit-$PHPUNIT_VERSION/phpunit";
 
     if (!file_exists($component = array_pop($argv))) {
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/PhpUnit/bin/simple-phpunit]: Can be removed

Comment (xabbuh) [inline src/Symfony/Bridge/PhpUnit/bin/simple-phpunit]: oops, good catch

**Comment by julienfalque** [comment]
@xabbuh I confirm this solves my issue too :)

**Comment by xabbuh** [comment]
@julienfalque Thanks! Happy new year to you! 🎉

**Comment by julienfalque** [comment]
Happy new year :) 🎉

**Comment by nicolas-grekas** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**